expertise/service/README.md

```diff
@@ -22,9 +22,26 @@ Returns code `200` on successful job submission with: `{"job_id": "string"}`\
 Returns code `400` if there was an error in the submitted config
 
 ## `GET /expertise/status`
-This endpoint gets the status of all jobs submitted by the user, or a single job with the given `job_id`. A valid request body has the following format:
+This endpoint gets the status of a single job with the given `job_id`. A valid request body has the following format:
 ```
-{"id": "string" [Optional]}
+{"id": "string"}
+```
+
+Returns the status, if any, that were submitted by the user and has the job id:
+```
+{
+	"job_id": "string",
```

Comment:
JSON fields should be camel-cased and should only contain underscores at the beginning of the field. You can read more about it [here](https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format)